cluster/ubuntu/util.sh

```diff
@@ -231,6 +231,11 @@ EOF
 }
 
 function create-kubelet-opts() {
+  if [ -n "$6" ] ; then
+      cni_opts=" --network-plugin=cni --network-plugin-dir=/etc/cni/net.d"
```

Comment:
For the maintainence reason mentioned above I would prefer that this worked with local-up-cluster as well. Otherwise this script, used by 2 people will drift and not work till someone on SO asks about it, and everyone will be clueless. I feel like this is relatively easy to combine, and local-up-cluster should work on any ubuntu localhost, but if it's harder than it looks feel free to defer.